src/server/api/errors.js

```diff
@@ -56,6 +56,28 @@ export async function assignmentRequired(user, assignmentId, assignment) {
   return true;
 }
 
+export async function assignmentAndNotSuspended(
```

Comment:
I think this should be built into the `assignmentRequired` function rather than have a separate function call.

just add the `accessRequired(user, organizationId, "TEXTER")` into there. The question is then where to get organizationId

 -- I guess in the past we've added it to the mutation as an argument (problematic, because then the texter could technically 'shop' sending a diff orgId to the server)
-- we could make sure looking up the org of the assignmentId is a fast cache-request, and just look it up (when unavailable in the higher context anyway) from the assignmentId
-- This would be worse for multi-tenant instances, but we could do something where suspension in one org basically suspends you from all orgs (in that case, the user-org affils are in-cache already, and we just have a isSuspended(user) function